main.tex

```diff
@@ -10,6 +10,8 @@
 
 \newcommand{\package}[1]{\texttt{#1}\xspace}
 \newcommand{\astroquery}{\package{astroquery}}
+\newcommand{\astropy}{Astropy\xspace}
```

Comment:
`\astropy` is currently not used. It was copied from Astropy paper v2.0. Feel free to remove if it is not needed. I left it in for now in case it is useful down the road.

Reply:
:+1: we will probably use it somewhere...